app/controllers/company_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CompanyController < ApplicationController  
+    def create_multiple
+        names = params["companies"]
+        result = []
+
+        names.each do |name|
```

Comment:
why not `result = names.map do |name| ...`?

Reply:
@jdguzman 
I fixed this one.